lab_2_retrieval_w_bm25/main.py

```diff
@@ -18,6 +23,12 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned.
     """
+    if not isinstance(text, str):
+        return None
+    for item in text:
+        if not item.isalpha() and item != ' ':
```

Comment:
use continue if item is alpha or whitespace

Reply:
Done, but can you explain why did I have to do this? I thought, that my previous code was normal. Maybe some new checks were needed and that's why I had to make new func?